Dockerfile

```diff
@@ -18,7 +18,7 @@ ADD setup.py requirements.txt /opt/atomicapp/
 # lets install pip, and gcc for the native extensions
 # and remove all after use
 RUN yum -y install epel-release && \
-    yum install -y --setopt=tsflags=nodocs python-pip python-setuptools docker gcc && \
+    yum install -y --setopt=tsflags=nodocs GitPython python-pip python-setuptools docker gcc && \
```

Comment:
I believe should keep gitpython within `requirements.txt` in order to keep this consistent across all Dockerfile's.
`gitpython==1.0.1` and prevent compatibility issues (Debian may have a different package) as seen with our previous lockfile issue.